groups/nts/ntsa/ntsa_ipv4address.h

```diff
@@ -58,99 +58,118 @@ class Ipv4Address
     Representation d_value;
 
   public:
-    enum {
+    /// Declare constants used by this implementation.
+    enum Constant {
+        /// The maximum required capacity of a buffer to store the longest
+        /// textual representation of an IPv4 addresses, not including the
+        /// null terminator.
         MAX_TEXT_LENGTH = 15
-        // The maximum required capacity of a buffer to store the longest
-        // textual representation of an IPv4 addresses, not including the
-        // null terminator.
     };
 
     /// Create a new IPv4 address having a default value.
-    Ipv4Address();
+    Ipv4Address() NTSCFG_NOEXCEPT;
+
+    /// Create a new IPv4 address having the same value as the specified
+    /// 'original' object. Assign an unspecified but valid value to the
+    /// 'original' original.
+    Ipv4Address(bslmf::MovableRef<Ipv4Address> original) NTSCFG_NOEXCEPT;
```

Comment:
I like what I see.
I think that it's worth to add some static assertions to the test driver:
`static_assert(bsl::is_nothrow_move_constructible_v<Ipv4Address>);`
Probably it can be written using bslmf stuff (so that no need to hide such static checks under a typedef)

Reply:
This is already tested in the test driver by checking `NTSCFG_TYPE_CHECK_BITWISE_MOVABLE`, which ensures the type is both bitwise-movable and  no-throw bitwise-constructible.